README.md

```diff
@@ -42,15 +56,9 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@ecekpo](https://github.com/ecekpo)
+- Twitter: [@ecekpo](https://twitter.com/ecekpo)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ecekpo)
 
 ## 🤝 Contributing
 
```

Comment:
![Screenshot from 2022-08-25 14-08-21](https://user-images.githubusercontent.com/93039359/186624556-0b6f1344-7e34-443d-ab9f-c9922ca5bd4a.png)
**[Optional]**
- [ ] Your link to your MIT.md file is broken. Please fix it as shown below :arrow_down:
`This project is [MIT](./MIT.md) licensed.`